### PR TITLE
Award years

### PR DESCRIPTION
**Pull request by Lathrisk**
Relates to #91 
Relates to #116

### CODE REVIEW

**Review by odscjames — DISMISSED** [review]

**Comment by odscjames** [comment]
12 is arbitary - I guess we can't just look in data to see what the earliest year to use is 'cos then the UI goes funny and inconsistent?

**Review by odscjames — DISMISSED** [review]

**Comment by Lathrisk** [comment]
@michaelwood @odscjames This is now ready for review

File: registry-vue/src/components/generic/YearsChart.vue [hunk 1]
```diff
@@ -14,6 +23,25 @@ export default {
...
+    return {
+      grantBandings: [0, 25, 50, 100, 250, 500, 1000, 5000],
+    };
+  },
+  methods: {
+    calculateOpacity(value){
+      const bandingsLength = this.grantBandings.length;
+
+      for (let i = 0; i < bandingsLength; i++) {
+        if (value == 0 || typeof value == "undefined" || value == null) {
```
Comment (michaelwood) [inline registry-vue/src/components/generic/YearsChart.vue:37]: Wondering if this is just (!value) here?

**Review by michaelwood — APPROVED** [review]
LGTM, do have a minor question on it but not a blocker

**Review by michaelwood — APPROVED** [review]

**Pull request merged**